.circleci/test-api.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+apt-get update
+apt-get install -y jq
```

Comment:
Installing dependencies should not be done on every test run, only in the dependencies step of the CI config 🙂
If this test is run on a local machine, for example, it might not have access to `apt-get`, or have `jq` already installed.